### DISCUSSION TITLE
Troubleshoot: Locked Out of ARM UI

### DISCUSSION THREAD

**Discussion by citizen-why** [automatic-ripping-machine/automatic-ripping-machine#651] [Q&A]
Somehow in a password migration i managed to get myself locked out of the ARM UI - i no longer have the username or password required to login. Fortunately I still have the login credentials for the arm process owner and can directly login into the server w/ sudo access. Is there a way to reset the UI login credentials. I've tried deleting the /opt/arm/installed file and revisiting http://[WEBSERVER_IP]:[WEBSERVER_PORT]/setup but get the message "You cannot create more than 1 admin account" and am still unable to login with the default login credentials.

Any thoughts / help would be appreciated.

**Comment by microtechno9000** [comment]
depending on what you have in your database and if you want to keep the record of completed jobs, there are two ways you could go about it.

1. Remove the ARM database, on restart it will recreate a new blank database
Note: this will cause you to loose all data from your database, only do this if you don't have any data you want to keep (i.e. completed job history).
To do this, stop the armui
`sudo systemctl stop armui.service `
remove the arm db file (may need to elevate priv, depending on how you have setup permissions)
`rm /home/arm/db/arm.db`
restart the armui
`sudo systemctl start armui.service `

3. Disable the login credentials in the ARM config file
To do this, stop the armui
`sudo systemctl stop armui.service `
edit the arm.yaml file, changing "DISABLE_LOGIN: false" to "DISABLE_LOGIN: true"
`sudo nano /etc/arm/config/arm.yaml`
restart the armui
`sudo systemctl start armui.service `

**Reply by citizen-why** [reply]
For some reason the DISABLE_LOGIN option is not being recognized and is requiring a login even when /etc/arm/config/arm.yaml ias "DISABLE_LOGIN: true".

Solution #1 worked well though
